### DISCUSSION TITLE
Where can I follow status for new language features and design thinking

### DISCUSSION THREAD

**Discussion by leandromoh** [dotnet/fsharp#11235] [Ideas]
I am a C# developer learning F#. 

C# language repository offers pages to follow [implementation status for language features](https://github.com/dotnet/roslyn/blob/main/docs/Language%20Feature%20Status.md) as well [Language Design Meetings](https://github.com/dotnet/csharplang/tree/main/meetings) and I think it is very cool because I know what is comming in next language versions, as well that language is not stagnant; there is a team working on it and giving visibility about it for community.

Does something similar for F#? I really miss it.

**Comment by NinoFloris** [comment]
You can find these under the @fsharp org :)

https://github.com/fsharp/fslang-suggestions
https://github.com/fsharp/fslang-design

**Reply by leandromoh** [reply]
Thanks, I found just approved suggestions and design stuff.
Where I find `implementation status for language features`?

**Reply by NinoFloris** [reply]
If you take a look at some of the RFCs you'll see a link to any implementation that may be in progress, if there is no link usually there is no implementation yet (though sometimes an implementation PR is open in dotnet/fsharp without the RFC being updated right away) 

Example:
https://github.com/fsharp/fslang-design/blob/master/RFCs/FS-1092-anonymous-type-tagged-unions.md
with implementation
https://github.com/dotnet/fsharp/pull/10896

**Reply by leandromoh** [reply]
Thanks @NinoFloris ! I gonna take a look.
